src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -71,6 +71,18 @@
             <argument type="service" id="templating.locator" />
         </service>
 
+        <service id="twig.extension.profiler" class="Symfony\Bridge\Twig\Extension\ProfilerExtension" public="false">
+            <argument type="service" id="twig.profile" />
+            <argument type="service" id="debug.stopwatch" on-invalid="ignore" />
+        </service>
+
+        <service id="twig.profile" class="Twig_Profiler_Profile" />
```

Comment:
this service is only meant to be accessed by ``twig.extension.profiler`` and ``data_collector.twig``. It does not need to be retrieved dynamically. Right ? If yes, it could be marked private